README.md

```diff
@@ -1,31 +1,201 @@
+
+```json
+{
+  "version": 1.2,
+  "tags": [
+    {
+      "name": "demo-wc-card",
+      "path": "./src/demo-wc-card.js",
+      "description": "This is a container looking like a card with a back and front side you can switch",
+      "attributes": [
+        {
+          "name": "header",
+          "type": "string",
+          "description": "Shown at the top of the card",
+          "defaultValue": "Your Message",
+          "required": true
+        }
+      ],
+      "properties": [
+        {
+          "name": "header",
+          "type": "string",
+          "description": "Shown at the top of the card",
+          "defaultValue": "Your Message",
+          "required": true
+        }
+      ],
+      "events": [
+        {
+          "name": "side-changed",
+          "description": "Fires whenever it switches between front/back"
+        }
+      ],
+      "methods": [
+        {
+          "name": "toggle",
+          "description": "Switches between front and back card"
+        }
+      ],
+      "slots": [
+        {
+          "name": "",
+          "description": "Content inside the card which displayed on the front page"
+        }
+      ],
+      "cssCustomProperties": [
+        {
+          "name": "--demo-wc-card-front-color",
+          "description": "Font color for the front",
+          "type": "color"
+        }
+      ],
+      "cssShadowParts": [
+        {
+          "name": "header",
+          "description": "Wrapper around the header part"
+        }
+      ]
+    }
+  ]
+}
+```
+
+#### Changelog
+
+- Added attribute details defaultValue [any]
+  - If no value is provided for the property this defaultValue will be used as a fallback
+- Added attribute details required [boolean]
+  - This property is required and the web component may throw an error if not provided
+- Added attribute details type [string]
+  - A type hint to indicate to the end-user what values can be provided
+- Added tag summary
+  - A markdown summary suitable for display in a listing
+- Added tag path
+  - Relative path to the js file which registers this web component
+- Added tag properties
+  - Lists all available properties of a web component
+- Added tag property details with name, description, default, required, type
+  - See attributes
+- Added tag methods with name, description
+  - Lists all methods or a web component
+  - Does not define parameters or type of parameters (this is for a later iteration) - for now, you can add this information to the description
+- Added tag events with name, description
+  - Lists all events the custom elements fires via `this.dispatchEvent()`
+- Added tag slots with name, description
+  - Lists all [slots](https://html.spec.whatwg.org/multipage/scripting.html#the-slot-element) in the shadow dom like `<slot name="header"></slot>`
+  - The default slot is represented via an empty name `<slot></slot>`
+- Added tag cssProperties with name, description, type
```

Comment:
This field is called `cssCustomProperties` in the schema and in the example above